Makefile

```diff
@@ -403,8 +405,6 @@ release-list:
 
 
 release:
-	@git commit --allow-empty -m "generating a new release"
-	@git push
```

Comment:
why was this removed? there was a reason for this change - without it you cannot tag repos without changes in them. for example - if you wanted to release some changes in accelerate but impact and directory should point to the latest master

Reply:
put this back here, it makes sense that both with an without a tag, we should be doing these.